resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -275,9 +287,13 @@ private[yarn] class YarnAllocator(
       s"executorsStarting: ${numExecutorsStarting.get}")
 
     if (missing > 0) {
-      logInfo(s"Will request $missing executor container(s), each with " +
-        s"${resource.getVirtualCores} core(s) and " +
-        s"${resource.getMemory} MB memory (including $memoryOverhead MB of overhead)")
+      var requestContainerMessage = s"Will request $missing executor container(s), each with " +
+          s"${resource.getVirtualCores} core(s) and " +
+          s"${resource.getMemory} MB memory (including $memoryOverhead MB of overhead)"
+      if (ResourceTypeHelper.isYarnResourceTypesAvailable()) {
+        requestContainerMessage ++= s" with custom resources: " + resource.toString
```

Comment:
should this extra messages instead be conditioned on `executorResourceTypes.nonEmpty`?  otherwise seems like it'll just repeat the memory & vcores from above in most cases.